jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2.java

```diff
@@ -180,24 +182,31 @@ private Path resolveExtraDirectory(MavenProject project) {
       if (pluginConfiguration != null) {
         Xpp3Dom extraDirectoryConfiguration = pluginConfiguration.getChild("extraDirectory");
         if (extraDirectoryConfiguration != null) {
-          Xpp3Dom child = extraDirectoryConfiguration.getChild("path");
-          if (child != null) {
+          Xpp3Dom pathChild = extraDirectoryConfiguration.getChild("path");
```

Comment:
this seems really intense, doesn't maven auto populate complex objects? Or is this because we support `<extraDirectories>/path</extraDirectories>`?

Should we phase/deprecate out the option to configure that?

Reply:
I believe you are talking about the way `FilesMojo` does with the `extraDirectory` field in it. I remember @TadCordle saying the reason for manual DOM traversal is because there was an issue in the multi-module case.

Reply:
Oh yeah I didn't realize this was FilesMojo.